addons/binding/org.openhab.binding.co2signal/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="co2signal"
```

Comment:
Please use tabs in XML files.

Reply:
Fixed but wondering why this isn't enforced in other bindings. I have yet to find a binding where all xml files are formatted the right way